apps/test/unit/templates/sectionAssessments/SubmissionStatusAssessmentsTableTest.js

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import {mount} from 'enzyme';
 import {assert, expect} from '../../../util/configuredChai';
```

Comment:
I think we are trying to move over to reconfiguredChai. While we are working on this test can you make that update? There are some examples in this PR: https://github.com/code-dot-org/code-dot-org/pull/27729/files#diff-659a5fd46cf16542e325da78e06765d7R3

Reply:
Good call - will update. Thanks!